python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -156,8 +170,27 @@ class DetectAndMeasureConfig(pipeBase.PipelineTaskConfig,
                  "base_PixelFlags_flag_interpolatedCenterAll",
                  "base_PixelFlags_flag_badCenterAll",
                  "base_PixelFlags_flag_edgeCenterAll",
+                 "base_PixelFlags_flag_saturatedCenterAll",
                  ),
     )
+    metricsMaskPlanes = lsst.pex.config.ListField(
+        dtype=str,
+        doc="List of mask planes to include in metrics",
+        default=('BAD', 'CLIPPED', 'CR', 'DETECTED', 'DETECTED_NEGATIVE', 'EDGE',
+                 'INEXACT_PSF', 'INJECTED', 'INJECTED_TEMPLATE', 'INTRP', 'NOT_DEBLENDED',
+                 'NO_DATA', 'REJECTED', 'SAT', 'SAT_TEMPLATE', 'SENSOR_EDGE', 'STREAK', 'SUSPECT',
+                 'UNMASKEDNAN',
+                 ),
+    )
+    metricSources = pexConfig.ConfigurableField(
+        target=SkyObjectsTask,
```

Comment:
I wonder if using sky objects as our probe will cause us to underestimate the mask fractions?  But as we discussed it's a reasonable place to start.